lib/mutant/mutator/node/named_value/access.rb

```diff
@@ -19,6 +19,44 @@ def dispatch
             emit_singletons
           end
 
+          # Named value access emitter for instance variables
+          class Ivar < Access
+
+            handle(:ivar)
+
+            children :name
+
+            # Emit mutations
+            #
+            # @return [undefined]
+            #
+            # @api private
+            def dispatch
+              emit_attribute
+              super()
+            end
+
+            private
+
+            # Emit instance variable as attribute send
+            #
+            # @return [undefined]
+            #
+            # @api private
+            def emit_attribute
+              emit(s(:send, nil, attribute_name))
+            end
+
+            # Variable name without leading '@'
+            #
+            # @return [Symbol]
+            #
+            # @api private
+            def attribute_name
+              name.slice(1..-1).to_sym
```

Comment:
Another nice side effect of moving literals to constants is you become more aware of their usage in code. Over-usage of string literals or other promitives in code is often a sign of other code smells being present, such as [Primitive Obsession] (http://c2.com/cgi/wiki?PrimitiveObsession) or [Stringly Typed](http://c2.com/cgi/wiki?StringlyTyped) coding.

Not saying that's the case here, just something to be aware of.